docs/UserGuide.md

```diff
@@ -423,11 +507,21 @@ Example Usage:
 snap zookeepbook_19-10-2020
 ```
 
+<p align="center"><img src="images/user-guide/beforesnap.png"/></p>
+
+<p align="center"><i>Figure 19: User entering snap command</i></p>
+
 Expected Outcome:
 ```
 Current ZooKeep Book saved as zookeepbook_19-10-2020.json
 ```
-A file named `zookeepbook_19-10-2020.json` is created in the `data/snapshots` directory.
+
+<p align="center"><img src="images/user-guide/aftersnap.png"/><img src="images/user-guide/snapresult.png"/></p>
+
+<p align="center"><i>Figure 20 (left): User executing snap command</i></p>
+<p align="center"><i>Figure 21 (right): The directory where the archived file is located</i></p>
```

Comment:
Not sure if I am seeing correctly but the preview does not display the 2 figures to be side by side.